mcs_benchmark_data/pipelines/commonsense_qa/commonsense_qa_benchmark_extractor.py

```diff
@@ -1,14 +1,30 @@
 from mcs_benchmark_data._extractor import _Extractor
 
+from mcs_benchmark_data.path import DATA_DIR_PATH
+
 
 class CommonsenseQaBenchmarkExtractor(_Extractor):
+    def __init__(
+        self, dev_jsonl_file: str, test_jsonl_file: str, train_jsonl_file: str, **kwds
```

Comment:
Prefer to say _name here, dev_jsonl_file_name, so it's clear it's not the full path.